ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -0,0 +1,76 @@
+import {faTimes} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {Button} from 'app/components/buttons';
+import {FlexColumn} from 'app/components/flex';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TextColumn} from 'app/components/text-column';
+import colors from 'app/styles/colors';
+import {withUserProfile} from 'app/utils';
+import {environment} from 'environments/environment';
+import {Profile} from 'generated/fetch';
+import * as React from 'react';
+
+export const CreateBillingAccountModal = withUserProfile() (
+  class extends React.Component<{
+    profileState: {profile: Profile},
+    onClose: Function
+  }, {}> {
+    constructor(props) {
+      super(props);
+    }
+
+    render() {
+      return <Modal width={600} onRequestClose={() => this.props.onClose()}>
+        <ModalTitle style={{marginBottom: '0.7rem'}}>Create a billing account</ModalTitle>
+
+        <ModalBody>
+          <FlexColumn style={{alignItems: 'flex-start'}}>
+            <img style={{width: '12rem', marginLeft: '-0.7rem'}} src='/assets/images/gcp_logo.png'/>
+
+            <TextColumn>
+              <div>Billing accounts are managed via Google Cloud Platform™ service.</div>
+              <div>Learn more on how to set up a billing account.</div>
+            </TextColumn>
+
+            <Button type='primary'
+                    style={{fontWeight: 400, padding: '0 18px', height: '40px'}}
+                    onClick={() => window.open(environment.createBillingAccountHelpUrl, '_blank')}>
+              Read Instructions
+            </Button>
+          </FlexColumn>
+        </ModalBody>
+
+        <hr style={{width: '100%', backgroundColor: colors.primary, borderWidth: '0px', height: '1px',
+          marginTop: '1rem',
+          marginBottom: '0.5rem'}}/>
+
+        <ModalFooter style={{marginTop: 0, justifyContent: 'flex-start'}}>
+          <FlexColumn>
+            <TextColumn>
+              <p style={{marginTop: 0}}>Don't already have a Google Cloud account?</p>
+              <a href='https://cloud.google.com' target='_blank'>Create billing account</a>
```

Comment:
Shouldn't these go to the Zendesk help URLs? If not, this seems like it should be the same URL as the other link below.